core/services/offchainreporting/logger.go

```diff
@@ -40,10 +44,18 @@ func (ol *ocrLogger) Warn(msg string, fields ocrtypes.LogFields) {
 }
 
 func (ol *ocrLogger) Error(msg string, fields ocrtypes.LogFields) {
```

Comment:
Having a Logger#Error function with a side effect other than simply logging might be a bit surprising, could we move this into a different abstraction?

Reply:
yeah it did feel a little strange, however introducing a new abstraction would require changing the libocr interface as far as I can tell. Not sure we want to block on that cc @spooktheducks